src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: no-repeat center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-of-type(-n+1),
+.stars--2 .stars__star:nth-of-type(-n+2),
+.stars--3 .stars__star:nth-of-type(-n+3),
+.stars--4 .stars__star:nth-of-type(-n+4),
+.stars--5 .stars__star:nth-of-type(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0 .stars__star:nth-of-type(n+1),
+.stars--1 .stars__star:nth-of-type(n+2),
+.stars--2 .stars__star:nth-of-type(n+3),
+.stars--3 .stars__star:nth-of-type(n+4),
+.stars--4 .stars__star:nth-of-type(n+5) {
```

Comment:
you don't have to write selectors for inactive stars, add them in stars__star, and selectors with star-active will rewrite them where needed